flac will fail pkgconf test pipeline

If the pkgconf test pipeline is added to flac it will fail in the following way:\r ```2024/11/12 23:10:41 INFO OCI layer digest: sha256:143ec1d07750c49d887e4e62ab551a757e82c9a260d4d5d346b1df57f41f247b
2024/11/12 23:10:41 INFO OCI layer diffID: sha256:b5f98705882abfaead05c8fa7e1bac5f17340e736db4937b9ca733e4951b726a
2024/11/12 23:10:41 INFO saving OCI image locally: apko.local/cache:d9f466ba0b7819b3505f937887eebde231e3875d669e27db53f8edad2151875c
2024/11/12 23:10:41 WARN skipping local domain tagging apko.local/cache:d9f466ba0b7819b3505f937887eebde231e3875d669e27db53f8edad2151875c as index.docker.io/library/melange:latest
2024/11/12 23:10:41 INFO ImgRef = apko.local/cache:d9f466ba0b7819b3505f937887eebde231e3875d669e27db53f8edad2151875c
2024/11/12 23:10:42 INFO running step "test/pkgconf"
2024/11/12 23:10:42 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:10:42 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:10:42 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:10:42 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:10:42 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN + basename /home/build/melange-out/flac-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN + dev_pkg=flac-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN + apk info -L flac-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN + grep -q ^Name: usr/lib/pkgconfig/flac++.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN + basename usr/lib/pkgconfig/flac++.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN + lib_name=flac++ uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN + echo usr/lib/pkgconfig/flac++.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN + grep -q '^usr/lib/pkgconfig/flac++.pc$\|^usr/share/pkgconfig/flac++.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN + pkgconf --exists flac++ uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN + grep -q ^Version: usr/lib/pkgconfig/flac++.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN + pkgconf --modversion flac++ uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 INFO 1.4.3 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN + grep -q ^Libs: usr/lib/pkgconfig/flac++.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN + pkgconf --libs flac++ uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 INFO -lFLAC++ -lFLAC uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN + grep -q ^Cflags: usr/lib/pkgconfig/flac++.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN + pkgconf --cflags flac++ uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN Package ogg was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN Perhaps you should add the directory containing `ogg.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 WARN Package 'ogg', required by 'flac', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:42 INFO pod b63bbd65e56a632db4e1924bc795406b1f0339adfcbdac34e9bc4b9055784237 terminated
2024/11/12 23:10:42 INFO pod 3387829c43b58314ee6c12e6a6e452bd3a5c30c5b64da02b70b317d8fda1df62 terminated
2024/11/12 23:10:42 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:10:42 INFO   workspace dir: /home/user/tmp/melange-workspace-3703482325
2024/11/12 23:10:42 INFO   guest dir: /home/user/tmp/melange-guest-3348494701
2024/11/12 23:10:42 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/flac] Error 1```

--- a/flac.yaml
+++ b/flac.yaml
@@ -2,7 +2,7 @@
 package:
   name: flac
   version: 1.4.3
-  epoch: 2
+  epoch: 3
   description: Free Lossless Audio Codec
   copyright:
     - license: BSD-3-Clause AND GPL-2.0-or-later
@@ -51,6 +51,12 @@ subpackages:
     pipeline:
       - uses: split/dev
     description: flac dev
+    dependencies:
+      runtime:
+        - libogg-dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: libflac
     pipeline:

